package.json

```diff
@@ -6,7 +6,7 @@
     "author": "hey@posthog.com",
     "license": "MIT",
     "scripts": {
-        "start": "pnpm build-rollup -w",
+        "start": "pnpm build-react && pnpm build-rollup -w",
```

Comment:
silly question maybe... this now builds react once and then builds rollup and starts watching just the rollup build

what's the benefit of building react here?

Reply:
React needs to exist, but it rarely if ever changes. So this issue was around running the next playground locally, it meant I had to call build first and then start which was simply annoying

Reply:
Ah, I'm a bad person and never use the playground 🙈